docusaurus/docs/adding-images-fonts-and-files.md

```diff
@@ -62,6 +62,6 @@ function App() {
 }
 ```
 
-This is handy if you don't want to load SVG as a separate file. Don't forget the curly braces in the import! The `ReactComponent` import name is significant and tells Create React App that you want a React component that renders an SVG, rather than its filename.
+This is handy if you don't want to load SVG as a separate file. Don't forget the curly braces in the import! The `ReactComponent` import name is significant and tells Create React App that you want a React component that renders an SVG, rather than its filename. Ensure PascalCasing of the component name (`Logo` in the above example).
```

Comment:
Nice, shall we reference or explain why PascalCasing should be used here? Just to ensure newcomers to React are brought up to speed :)

Reply:
I'd really love that. I have myself started learning React a few days ago and as a newcomer had to do some trial and error before reaching this conclusion. I went through the docs and feel [choosing-the-type-at-runtime](https://reactjs.org/docs/jsx-in-depth.html#choosing-the-type-at-runtime) would be the closest match as a reference for our use case. Please provide your inputs.